README.md

```diff
@@ -1,78 +1,83 @@
-![Logo Banner](./docs/images/kuwala_title_banner.png)
-
 [![Slack](https://img.shields.io/badge/slack-chat-orange.svg)](https://join.slack.com/t/kuwala-community/shared_invite/zt-l5b2yjfp-pXKFBjbnl7_P3nXtwca5ag)
 ![License](https://img.shields.io/github/license/kuwala-io/kuwala)
 
-## What is Kuwala?
+<img src="./docs/images/kuwala_header_github.png" width="100%" />
+
+<p align="center">
+  <a href="https://join.slack.com/t/kuwala-community/shared_invite/zt-l5b2yjfp-pXKFBjbnl7_P3nXtwca5ag"><img src="./docs/images/slack_join_button_github.png" height="96" /></a>
+  <a href="https://github.com/orgs/kuwala-io/projects/3/views/6"><img src="./docs/images/roadmap_button_github.png" height="96" /></a>
+</p>
 
-Kuwala is a tool to build rich features for analytics based on clean data. It uses 
-[PySpark](http://spark.apache.org/docs/latest/api/python/) in combination with 
-[Parquet](http://parquet.apache.org/documentation/latest/) for data processing. The resulting clean datasets are then
-loaded into a [Postgres](https://www.postgresql.org) data warehouse. We then use [dbt](https://www.getdbt.com) for fast, 
-flexible, and reproducible feature generation.
+Kuwala is the no-code data platform for BI analysts and engineers enabling you to build powerful analytics workflows. We 
+are set out to bring state-of-the-art data engineering tools you love, such as 
```

Comment:
are set out to bring **state-of-the-art data engineering tools** you love, such as